terraform/modules/upload-service/bucket.tf

```diff
@@ -85,4 +89,7 @@ resource "aws_s3_bucket" "lambda_deployments" {
   acl = "private"
   force_destroy = "false"
   acceleration_status = "Enabled"
+  tags = {
```

Comment:
Can we also add the Owner, Service and and environment (if applicable) tags like elsewhere?

Reply:
Added in config applied the changes, tags should be added in the resources now.